Auth.php

```diff
@@ -32,6 +37,7 @@ public function __construct(Model $userModel = null)
         }
 
         $this->userModel = $userModel;
+        $this->fallbackAuth = new \Piwik\Plugins\Login\Auth();
```

Comment:
I think no matter what, Login\Auth will have to obey the Auth interface's contract, so shouldn't be an issue even if it changes. Only if Auth changes, but that's to be expected. I could be wrong though...

Anyway, will merge ;)